helm/chaos-mesh/values.yaml

```diff
@@ -249,7 +249,7 @@ dashboard:
         tlsSecret: dashboard.local-tls
 
 dnsServer:
-  create: false
+  create: true
```

Comment:
It changed the default behavior of the chart installation. Why do we need to update this line inside this PR?